backend/app/server/routes/post.py

```diff
@@ -37,10 +37,10 @@ async def delete_post_data(post_id: str = Body(..., embed=True), current_user=De
 
 
 @router.patch("/update", response_description="Update post")
-async def update_post_data(post_id: str, updated_post: UpdatePostModel = Body(...), current_user=Depends(auth_handler.auth_wrapper)):
+async def update_post_data(updated_post: UpdatePostModel = Body(...), current_user=Depends(auth_handler.auth_wrapper)):
     updated_post = jsonable_encoder(updated_post)
-    new_post = await update_post(current_user, post_id, updated_post)
-    return ResponseModel(new_post, "Post updated successfully.")
+    new_post = await update_post(current_user, updated_post["post_id"], updated_post)
+    return ResponseModel(new_post,"Post updated successfully.")
```

Comment:
I'm basically doing a Flake8 manual check atm. xD
```suggestion
    return ResponseModel(new_post, "Post updated successfully.")
```